README.md

```diff
@@ -149,6 +149,10 @@ The base model class is a MultiTaskModel. To add another model, first add the cl
 Task-specific components include logistic regression and multi-layer perceptron for classification and regression tasks, and an RNN decoder with attention for sequence transduction tasks.
 To see the full set of available params, see [config/defaults.conf](config/defaults.conf). For a list of options affecting the execution pipeline (which configuration file to use, whether to enable remote logging or tensorboard, etc.), see the arguments section in [main.py](main.py).
 
+### Supported Functionality
+
+We allow for training on a shared encoder from the word contextual embeddings, as well as forwarding from the embeddings to the task specific module (by setting sent_enc = null).  Additionally, if you would like to add a skip connection from the contextual embedding directly to the module, set skip_embs = 1. 
```

Comment:
High-level comment: It's fairly confusing that two of our basic use-cases are so different. To change from BERT+classifier to randomly-initialized-BiLSTM+classifier, you need to change quite a lot, since they don't play the same role in our pipeline. Do whatever you can to make the setup clear, including renaming variables where that can help, and adding explicit reminders in several places that ELMo/BERT/GPT are all treated as part of the input handler and aren't shared sentence encoders in our sense.

Reply:
Up to you how much of this belongs in this PR.